drivers/auxdisplay/Kconfig.serlcd

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2023 Jan Henke <Jan.Henke@taujhe.de>
+# SPDX-License-Identifier: Apache-2.0
+
+config AUXDISPLAY_SERLCD
+	bool "SparkFun SerLCD dot character VFD driver"
```

Comment:
I had a search for "SparkFun SerLCD" and found the product page: https://www.sparkfun.com/categories/tags/serlcd which shows LCDs, not VFDs, so if these is not a VFD then please update. Also this suggests that the _is_ a backlight on these but this driver does not support it, are you planning to add support for the backlight in the future?

Reply:
Indeed, c&p error.

Regarding the backlight, I did not get the backlight control to work properly yet. So I would like to merge it first without the backlight and add it later, once it works here.